plugin-server/src/main/ingestion-queues/batch-processing/each-batch-async-handlers.ts

```diff
@@ -62,6 +66,29 @@ export async function eachBatchAppsOnEventHandlers(
     await eachBatch(payload, queue, eachMessageAppsOnEventHandlers, groupIntoBatches, 'async_handlers_on_event')
 }
 
+function groupIntoBatchesWebhooks(actionMatcher: ActionMatcher) {
```

Comment:
We already have a simple `groupIntoBatches` in utils, I think it would be good for maintainability to have both functions live in the same file